contracts/samples/gnosis/EIP4337Manager.sol

```diff
@@ -55,11 +55,6 @@ contract EIP4337Manager is IAccount, GnosisSafeStorage, Executor {
             validationData = SIG_VALIDATION_FAILED;
         }
 
```

Comment:
Shouldn't `validateUserOp` require `key==0` to enforce 1D nonce and preserve the previous Safe behavior?

Reply:
Are you adding that?

Reply:
Look at the code. It is there